lib/rules/padding-line-between-tags.js

```diff
@@ -109,40 +140,38 @@ function checkNewline(context) {
         const lineDifference =
           closestSibling.loc.start.line - endTag.loc.end.line
         if (configure.blankLine === 'always') {
-          if (lineDifference === 1) {
-            insertNewLine(context, block, closestSibling)
-          } else if (lineDifference === 0) {
-            context.report({
-              messageId: 'always',
-              loc: closestSibling.loc,
-              // @ts-ignore
-              fix(fixer) {
-                const lastSpaces = /** @type {RegExpExecArray} */ (
-                  /^\s*/.exec(
-                    context.getSourceCode().lines[endTag.loc.start.line - 1]
-                  )
-                )[0]
-
-                return fixer.insertTextAfter(endTag, `\n\n${lastSpaces}`)
-              }
-            })
-          }
-        } else {
-          if (lineDifference > 1) {
-            let hasOnlyTextBetween = true
-            for (
-              let i = endTag.loc.start.line;
-              i < closestSibling.loc.start.line - 1 && hasOnlyTextBetween;
-              i++
-            ) {
-              hasOnlyTextBetween = !/^\s*$/.test(
-                context.getSourceCode().lines[i]
-              )
+          insertNewLine(context, block, closestSibling, lineDifference)
+        } else if (configure.blankLine === 'consistent') {
```

Comment:
I think we need to look more into how the `consistent` option works.
For example, I think the following case should not be warned.

```vue
<template>
  <div>
    <div />

    <div />

    <br/>
    <br/>
    <br/>
    <br/>
    <br/>
    <br/>

    <div />

    <div />
  </div>
</template>

<script>
/* eslint   "vue/padding-line-between-tags": ["error", [
  { "blankLine": "consistent", "prev": "*", "next": "*" },
  { "blankLine": "never", "prev": "br", "next": "br" },
]] */
</script>
```

What do you think?

Reply:
Hi, I'm not sure what you mean by this, this template will not produce any errors

Reply:
Nevermind, I reran the test and it failed :)